### DISCUSSION TITLE
Install motionplus on Ubuntu 24.04 (or beta) - temp fix

### DISCUSSION THREAD

**Discussion by axeljerabek** [Motion-Project/motionplus#142] [Q&A]
I was struggling to compile and install motionplus on the new Ubuntu 24.04 (and beta) but have a temporary fix if anyone runs into the same problem after upgrading the 22.04 LTS version:

You need to install 3 libs from the old Ubuntu 22.04 LTS Jammy release: libopenexr25, libsrt1.4-openssl and libsvtav1enc0.

edit /etc/apt/sources.list
and add the line
"deb http://cz.archive.ubuntu.com/ubuntu jammy main universe"
to install the 3 libs from the old 22.04 Jammy Ubuntu

then type: 
apt update

and

apt install libopenexr25
apt install libsrt1.4-openssl
apt install libsvtav1enc0

motionplus should work again.

**Comment by Arunkumar-dev-0** [comment]
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package libsvtav1enc0 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package libopenexr25
E: Unable to locate package libsrt1.4-openssl
E: Couldn't find any package by glob 'libsrt1.4-openssl'
E: Package 'libsvtav1enc0' has no installation candidate


Not working motionpro

**Comment by Mr-Dave** [comment]
@Arunkumar-dev-0 If you followed the original notes you will need to undo all those changes or delete the system and start with a fresh install of 22.04.  You must get to the point of having 

` sudo apt update` `sudo apt dist-upgrade` `sudo apt autoremove` 

all showing no errors and nothing to upgrade or remove. 

(I created a fresh install of 22.04 today and it does not show any of the package problems that are listed.)
